var/spack/repos/builtin/packages/util-linux-uuid/package.py

```diff
@@ -19,6 +19,7 @@ class UtilLinuxUuid(AutotoolsPackage):
     version('2.36',   sha256='82942cd877a989f6d12d4ce2c757fb67ec53d8c5cd9af0537141ec5f84a2eea3')
 
     conflicts('%gcc@:4', when='@2.37:')
+    conflicts('%intel', when='@2.37.4')
```

Comment:
Why conflicting with a specific version? Do v2.36 and v2.36.2 build with intel?

Reply:
Yes, they do.

Reply:
And 2.38 is expected to? Is it known to conflict with all intel compiler versions?